frontend/src/components/Common/SignIn/SignIn.tsx

```diff
@@ -8,7 +8,9 @@ import { auth, provider } from '../../../firebase';
 
 const SignIn: React.FC = () => {
   const onGoogleSignIn = () => {
-    signInWithPopup(auth, provider);
+    signInWithPopup(auth, provider).catch((err) => {
+      console.error(err);
```

Comment:
Did you mean to put this change in here? If so, it's probably small enough that it's not a big deal that it's unrelated. But worth noting in the PR description.

Reply:
updated PR description. TLDR: there was an error that shows up when you close the sign-in pop-up without signing in, however, there shouldn't be any logic change just because that error is thrown.

Reply:
* I changed it so theres no console logging for it. I think the catch will still be necessary. Since the user shouldn't see the log that they canceled a popup.